prototype/distribution/features/northbound-ogc-sensorthings-mqtt-feature/src/main/resources/northbound-ogc-sensorthings-mqtt-feature.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "org.eclipse.sensinact.gateway.distribution.features:northbound-ogc-sensorthings-mqtt-feature:osgifeature:0.0.2-SNAPSHOT",
+  "bundles": [
+    { "id": "io.moquette:moquette-broker:0.16" },
+    { "id": "io.netty:netty-common:4.1.77.Final" },
+    { "id": "io.netty:netty-buffer:4.1.77.Final" },
+    { "id": "io.netty:netty-codec:4.1.77.Final" },
+    { "id": "io.netty:netty-codec-http:4.1.77.Final" },
+    { "id": "io.netty:netty-codec-mqtt:4.1.77.Final" },
+    { "id": "io.netty:netty-common:4.1.77.Final" },
+    { "id": "io.netty:netty-handler:4.1.77.Final" },
+    { "id": "io.netty:netty-resolver:4.1.77.Final" },
+    { "id": "io.netty:netty-transport:4.1.77.Final" },
+    { "id": "io.netty:netty-transport-classes-epoll:4.1.77.Final" },
+    { "id": "io.netty:netty-transport-native-unix-common:4.1.77.Final" },
+    { "id": "org.eclipse.sensinact.gateway.northbound.sensorthings:mqtt:0.0.2-SNAPSHOT" }
+],
+  "extensions": {
+    "sensinact.feature.depends": {
+      "kind": "mandatory",
+      "type": "artifacts",
+      "artifacts": [
+        {
+          "id": "org.eclipse.sensinact.gateway.distribution.features:core-feature:osgifeature:0.0.2-SNAPSHOT"
+        },
+        {
+          "id": "org.eclipse.sensinact.gateway.distribution.features:northbound-ogc-sensorthings-feature:osgifeature:0.0.2-SNAPSHOT"
```

Comment:
I don't think that this dependency is needed. The MQTT feature is deployable separately from the REST feature.